cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -3946,4 +3947,28 @@ private boolean shouldAutoRequestNotificationPermission() {
   public void setAutoRequestNotificationPermission(boolean autoRequestNotificationPermission) {
     this.autoRequestNotificationPermission = autoRequestNotificationPermission;
   }
+
+  public int getBadgeCount() {
+    try {
+      int badgeCount = 0;
+      if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.M) {
+        StatusBarNotification[] activeNotifications = BadgeHelper.getActiveNotifications(context);
+
+        for (StatusBarNotification activeNotification : activeNotifications) {
```

Comment:
Please put the logic inside BadgeHelper and here just call BadgeHelper.getBadgeCount() e.g.